pipeline/test/test_pipeline.py

```diff
@@ -257,6 +256,30 @@ def test_add_extension_input_resource_file(self):
             in1.add_extension('.baz')
         assert in1._value.endswith('.txt.bgz.foo')
 
+    def test_gcs_file_localization(self):
+        p = Pipeline()
```

Comment:
I'm not sure why this test isn't in master already. Should have gone in on a different branch.

Reply:
Forgot I made an issue to fix this #5767 once user accounts are in pipeline.